src/main/java/com/commercetools/sync/categories/utils/CategorySyncUtils.java

```diff
@@ -89,7 +89,9 @@
     @Nonnull
     private static List<UpdateAction<Category>> buildUpdateActionsFromOptionals(
         @Nonnull final List<Optional<UpdateAction<Category>>> optionalUpdateActions) {
-        return asList(optionalUpdateActions.stream());
+        return optionalUpdateActions.stream()
```

Comment:
Refer to this comment: https://github.com/commercetools/commercetools-sync-java/pull/275/files#diff-93cb8feb4d531930ae6babf9eb6b4c4fR68